kerl

```diff
@@ -733,6 +743,63 @@ do_deploy()
     echo "kerl_deactivate"
 }
 
+is_valid_install_path()
+{
+    # don't allow installs into home directory
+    if [ "$1" = "$HOME" ]; then
+        echo "ERROR: You cannot install a build into $HOME. It's a really bad idea."
+        return 1
+    fi
+
+    # don't allow installs into .erlang because
+    # it's a special configuration file location
+    # for OTP
+    if [ "$1" = "$HOME/.erlang" ]; then
+        echo "ERROR: You cannot install a build into $HOME/.erlang. (It's a special configuration file location for OTP.)"
+        return 1
+    fi
+
+    # don't install into .kerl either.
+    if [ "$1" = "$HOME/.kerl" ]; then
```

Comment:
```
mallen-rMBP:~ mallen$ readlink -f .
readlink: illegal option -- f
```

Great suggestion. Seems to be the common suggestion from searching but doesn't work on OS X.